rotkehlchen/db/dbhandler.py

```diff
@@ -2186,6 +2187,7 @@ def add_exchange(
             api_secret: ApiSecret,
             passphrase: Optional[str] = None,
             kraken_account_type: Optional[KrakenAccountType] = None,
+            binance_markets: Optional[List[str]] = None,  # pylint: disable=W0613
```

Comment:
Ehmm ... why are you ignoring the unused argument from pylint? Shouldn't you actually use the `binance_markets` argument in this function to save it in the DB?